lib/cluster.js

```diff
@@ -427,8 +442,8 @@ function masterInit() {
       queryServer(worker, message);
     else if (message.act === 'listening')
       listening(worker, message);
-    else if (message.act === 'suicide')
```

Comment:
since the message should only be internal, I don't think so. Unless you or someone else feel differently about it